sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -195,9 +202,28 @@ case class Aggregate(
     child: LogicalPlan)
   extends UnaryNode {
 
+  override lazy val resolved: Boolean = {
+    val hasWindowExpressions = aggregateExpressions.exists ( _.collect {
+        case window: WindowExpression => window
+      }.nonEmpty
+    )
+
+    !expressions.exists(!_.resolved) && childrenResolved && !hasWindowExpressions
+  }
+
   override def output: Seq[Attribute] = aggregateExpressions.map(_.toAttribute)
 }
 
+case class Window(
+    projectList: Seq[Attribute],
```

Comment:
@cloud-fan I found the reason. We still need `projectList `. When we have an expression in window specification, the `projectList ` also contains the alias names. 

For example, 

```SQL 
select key, value,
sum(value) over (partition by key % 2 order by key) as sum
from parquet_t1 group by key, value
```
```
Project [key#19L,value#20,sum#18]
+- Project [key#19L,value#20,_w0#24,_w1#25L,sum#18,sum#18]
   +- Window [key#19L,value#20,_w0#24,_w1#25L], [(sum(_w0#24),mode=Complete,isDistinct=false) windowspecdefinition(_w1#25L, key#19L ASC, RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) AS sum#18], [_w1#25L], [key#19L ASC]
      +- Aggregate [key#19L,value#20], [key#19L,value#20,cast(value#20 as double) AS _w0#24,(key#19L % cast(2 as bigint)) AS _w1#25L]
         +- SubqueryAlias parquet_t1
            +- Relation[key#19L,value#20] ParquetRelation
```